test/tokenbridge.test.ts

```diff
@@ -153,193 +182,229 @@ describe('TokenBridge', async function() {
-    expect(await l1tokenUsdc.isImplementation()).to.be.false;
-    expect(await l1tokenUsdcImplementation.isImplementation()).to.be.true;
-
-    expect(await l1tokenDai.INCENTIVES_CONTROLLER()).to.eq(INCENTIVES_CONTROLLER);
-    expect(await l1tokenDai.LENDING_POOL()).to.eq(LENDING_POOL);
-    expect(await l1tokenDai.ATOKEN()).to.eq(A_DAI);
-    expect(await l1tokenDai.ASSET()).to.eq(await aDai.UNDERLYING_ASSET_ADDRESS());
-    expect(await l1tokenDai.REWARD_TOKEN()).to.eq(rewAaveTokenL1.address);  
+    const daiInitArgs = [pool.address, aDai.address, "Wrapped aDAI", "waaDAI", l1TokenBridgeProxy.address];
+    l1StaticDai = await initStaticATokenProxy(l1StaticDaiImpl.address, l1StaticDaiProxy, daiInitArgs);
+    expect(await l1StaticDai.isImplementation()).to.be.false;
+    expect(await l1StaticDaiImpl.isImplementation()).to.be.true;
+    const usdcInitArgs = [pool.address, aUsdc.address, "Wrapped aUSDC", "waaUSDC", l1TokenBridgeProxy.address];
+    l1StaticUsdc = await initStaticATokenProxy(l1StaticUsdcImpl.address, l1StaticUsdcProxy, usdcInitArgs);
+    expect(await l1StaticUsdc.isImplementation()).to.be.false;
+    expect(await l1StaticUsdcImpl.isImplementation()).to.be.true;
+
+    expect(await l1StaticDai.INCENTIVES_CONTROLLER()).to.eq(INCENTIVES_CONTROLLER);
+    expect(await l1StaticDai.LENDING_POOL()).to.eq(LENDING_POOL);
+    expect(await l1StaticDai.ATOKEN()).to.eq(A_DAI);
+    expect(await l1StaticDai.ASSET()).to.eq(await aDai.UNDERLYING_ASSET_ADDRESS());
+    expect(await l1StaticDai.REWARD_TOKEN()).to.eq(l1AAVE.address);  
   })
 
-  it('l1user receives tokens and converts them to staticATokens', async () => {
-    // l1user receives tokens
-    await dai.connect(daiWhale).transfer(l1user.address, 1000);
-    await usdc.connect(usdcWhale).transfer(l1user.address, 1000);
-    await rewAaveTokenL1.connect(stkaaveWhale).transfer(signer.address, 1000);
-    // l1user deposits dai and usdc and gets corresponding staticATokens
-    await dai.connect(l1user).approve(l1tokenDai.address, MAX_UINT256);
-    await l1tokenDai.connect(l1user).deposit(l1user.address, 1000, 0, true);
-    for (let i = 0; i < 5; i++) {
-      await ethers.provider.send('evm_mine', []);
-    }
-    await usdc.connect(l1user).approve(l1tokenUsdc.address, MAX_UINT256);
-    await l1tokenUsdc.connect(l1user).deposit(l1user.address, 1000, 0, true);
+  it('l1user receives tokens and converts them to aTokens and staticATokens', async () => {
+    // l1user receives dai and usdc
+    await dai.connect(daiWhale).transfer(l1user.address, 3000);
+    await usdc.connect(usdcWhale).transfer(l1user.address, 3000);
+
+    // l1user deposits dai and gets aDai
+    await dai.connect(l1user).approve(pool.address, MAX_UINT256);
+    await pool.connect(l1user).deposit(dai.address, 1000, l1user.address, 0);
+
+    // l1user deposits dai and gets staticADai
+    await dai.connect(l1user).approve(l1StaticDai.address, MAX_UINT256);
```

Comment:
that line is no longer needed because of [this one](https://github.com/aave-starknet-project/aave-starknet-bridge/pull/32/files/766eb086430dafbb50bffd5d9f4a0933b883e3f3..184db62c07c796628019f9dc1b5ae031eb174b81#diff-82d373285757d7293feb043340e157af235cca879cc09a3900b5db6fbda67d18R100), right?